WooCommerce/Classes/ViewRelated/Orders/Order Creation/ProductsSection/AddProductToOrder.swift

```diff
@@ -25,8 +25,8 @@ struct AddProductToOrder: View {
                             Divider().frame(height: Constants.dividerHeight)
                                 .padding(.leading, Constants.defaultPadding)
                         }
+                        .background(Color(.listForeground))
```

Comment:
This change is not specifically related to the new error notices, but while testing I noticed that the infinite scroll loading indicator had the wrong background color. This fixes it by not applying the background color to the whole infinite scroll list, but only the product rows in the list.